### PR TITLE
Add CANINE and TAPAS to doc tests

### PR DESCRIPTION
**Pull request by NielsRogge**
# What does this PR do?

Adds CANINE and TAPAS to the doc tests.

### CODE REVIEW

**Comment by ydshieh** [comment]
Thank you working on this, @NielsRogge 

Do you think it's possible to:
- Add more realistic examples for `TapasForQuestionAnswering`, `TapasForSequenceClassification`, etc.
- Add expected output shape for `TapasModel` and make its docstring output the shape instead of the logits.

@patrickvonplaten Do you think it is good to (always) ask the contributors to change the docstrings of `ModelForXXX` to something more meaningful (i.e. instead of the logits), as my comments here above?

File: src/transformers/models/canine/modeling_canine.py [hunk 1]
```diff
@@ -52,6 +52,16 @@
 _CONFIG_FOR_DOC = "CanineConfig"
 _TOKENIZER_FOR_DOC = "CanineTokenizer"
 
+# Base model docstring
+_EXPECTED_OUTPUT_SHAPE = [1, 8, 768]
```
Comment (patrickvonplaten) [inline src/transformers/models/canine/modeling_canine.py:56]: Hey Niels, sorry we changed the template just today since just testing for shapes doesn't really make sense IMO. Could you adapt the expected values according to https://github.com/huggingface/transformers/pull/16363/files#r833804669

Wrote a guide here: https://github.com/huggingface/transformers/issues/16292#issuecomment-1076919075 on how to find good checkpoins as well. If Canine doesn't have any good fine-tuned checkpoints for a task, I'd recommend to just upload one with a random head and use this to make the test not fail due to random values

Comment (patrickvonplaten) [inline src/transformers/models/canine/modeling_canine.py:56]: Hey Niels, sorry we changed the template just today since just testing for shapes doesn't really make sense IMO. Could you adapt the expected values according to https://github.com/huggingface/transformers/pull/16363/files#r833804669

Wrote a guide here: https://github.com/huggingface/transformers/issues/16292#issuecomment-1076919075 on how to find good checkpoins as well. If Canine doesn't have any good fine-tuned checkpoints for a task, I'd recommend to just upload one with a random head and use this to make the test not fail due to random values

Comment (ydshieh) [reply src/transformers/models/canine/modeling_canine.py:56]: To avoid confusion, @patrickvonplaten  is talking about the `...ForXXX` models for downstream tasks, not the base model.

File: utils/documentation_tests.txt [hunk 2]
```diff
@@ -49,3 +49,17 @@ src/transformers/models/wav2vec2/modeling_wav2vec2.py
...
+src/transformers/models/pegasus/modeling_pegasus.py
+src/transformers/models/blenderbot/modeling_blenderbot.py
+src/transformers/models/blenderbot_small/modeling_blenderbot_small.py
+src/transformers/models/plbart/modeling_plbart.py
+src/transformers/generation_utils.py
+docs/source/quicktour.mdx
+docs/source/task_summary.mdx
+src/transformers/models/resnet/modeling_resnet.py
+src/transformers/models/glpn/modeling_glpn.py
+src/transformers/models/canine/modeling_canine.py
```
Comment (ydshieh) [inline utils/documentation_tests.txt:65]: Hi, @NielsRogge 

I know this PR is still in draft. But do you happen to know why there are these many lines added which are mostly duplicated.

This also happened in your previous PR (#16199).
